specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ManagedEnvironments.json

```diff
@@ -501,24 +828,167 @@
           "x-ms-secret": true
         }
       }
-    }
-  },
-  "securityDefinitions": {
-    "azure_auth": {
-      "type": "oauth2",
-      "description": "Azure Active Directory OAuth2 Flow",
-      "flow": "implicit",
-      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
-      "scopes": {
-        "user_impersonation": "impersonate your user account"
+    },
+    "Certificate": {
+      "description": "Certificate used for Custom Domain bindings of Container Apps in a Managed Environment",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of the Certificate.",
+          "enum": [
+            "Succeeded",
+            "Failed"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CertificateProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "password": {
```

Comment:
I think this should be marked as secret and write-only.

Reply:
Yes. Will add `"x-ms-secret": true`. However, I think it is already write-only: https://github.com/Azure/azure-rest-api-specs/blob/3fce21cd24251a1cf699f031ea69826e9d7a251d/specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ManagedEnvironments.json#L857-L859